gammapy/irf/effective_area_table.py

```diff
@@ -360,8 +364,24 @@ class EffectiveAreaTable2D(object):
     >>> from gammapy.datasets import load_aeff2D_fits_table
     >>> aeff2D = EffectiveAreaTable2D.from_fits(load_aeff2D_fits_table())
     >>> offset = Angle(0.6, 'degree')
-    >>> energies = Quantity(np.logspace(0, 1, 60), 'TeV')
-    >>> eff_area = aeff2D.evaluate(offset, energies)
+    >>> energy = Quantity(np.logspace(0, 1, 60), 'TeV')
+    >>> eff_area = aeff2D.evaluate(offset, energy)
+
+    Create ARF fits file for a given offest and energy binning:
+
+    >>> import numpy as np
+    >>> from astropy.coordinates import Angle
+    >>> from astropy.units import Quantity
+    >>> from gammapy.irf import EffectiveAreaTable2D
+    >>> from gammapy.datasets import load_aeff2D_fits_table
+    >>> aeff2D = EffectiveAreaTable2D.from_fits(load_aeff2D_fits_table())
+    >>> offset = Angle(0.43, 'degree')
+    >>> nbins = 50
+    >>> energy = Quantity(np.logspace(0, 1, nbins+1), 'TeV')
```

Comment:
@kingj90 – I don't understand what your preference is. You say "get rid of energy_band tuple" and "get rid of energy_lo and energy_hi", but as I understand it these are the two options?

I don't think Quantity or not is an issue ... in Gammapy energies should always be quantities and then when reading / writing to file we just have to make sure we store the unit correctly (or convert to an agreed unit if the storage format doesn't support storing the unit).

@kingj90 - it might make sense to just do here whatever you like best and split this discussion out into a separate issue / pull request. As done for times [here](https://gammapy.readthedocs.org/en/latest/time/index.html#time-handling-in-gammapy) we should have a "How to handle energies" section in the docs that also explains how to handle energy bounds, and the PR where this is written could then also adjust the API to be more consistent.